src/App.js

```diff
@@ -45,10 +45,18 @@ class Navbar extends React.Component {
 class SelectionBox extends React.Component {
   render() {
     return (
-      <div className="selectionBox">
-       <div> CollegePrep is here to help you with all you need to improve your score. </div>
+    <div className="selectionBox">
+       <div id="bold-item"> {this.props.heading} </div>
+        <div>{this.props.p}</div>
+        <br />
+        <div className="testimony">
```

Comment:
Let's make its own component, so that we can reuse the testimony elsewhere where we don't have a selection box.